### PR TITLE
Lm bunch css + new home page

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
The GWT logo is wrong, right?

**Comment by LarryMatte** [comment]
My bad... that file should have been deleted

File: site/src/main/site/assets/less/style.less [hunk 1]
```diff
@@ -314,31 +339,119 @@ aside#submenu {
...
+  a#logo-mobile {
+    @media(@mobile) {
+      background: url('../img/gwtpLogo.png') center 1.5rem no-repeat @c-gwtp;
+      -webkit-background-size: contain;
+      background-size: contain;
+    }
   }
 }
 
+//a#logo-mobile {
```
Comment (spg) [inline site/src/main/site/assets/less/style.less]: remove commented code

**Comment by spg** [comment]
@jasonlemay pls review :)

**Pull request merged**